README.md

```diff
@@ -211,7 +211,7 @@ TARGET SELECTION:
         -F, --force-no-reboot
             Force a device not in BOOTSEL mode but running compatible code to reset so the command can be executed. After executing the
             command (unless the command itself is a 'reboot') the device will be left connected and accessible to picotool, but without the
-            RPI-RP2 drive mounted
```

Comment:
I dunno if it's "technically correct" but in some other places I refer to this as the "BOOTSEL drive", which _might_ be less ambiguous than "USB drive"? :shrug: